lua/lspconfig/volarvuels.lua

```diff
@@ -0,0 +1,85 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'volarvuels'
+local bin_name = 'vue-server'
```

Comment:
`vue-server` executable is only exposed in @volar/server@alpha so it may change in future releases. Let's wait a bit.

Reply:
It's called `volar-server` now in latest alpha.

Reply:
Sounds like this will be the bin name for a while (based on maintainer's comments)